pkg/driver/driver.go

```diff
@@ -12,6 +12,8 @@ type Driver interface {
 
 	CreateDisk() error
 
+	GetPIDName() string
```

Comment:
```suggestion
	Name() string
```

The pid file name can be just `Name() + ".pid"`

Reply:
Ah nice !! This looks more clean.

Done the changes